R/comparePlot.R

```diff
@@ -107,23 +91,19 @@
 #'
 #' @export
 comparePlot <- function(compareDF,
+                        plotType = "canvasXpress",
                         pThreshold = 0.01,
                         xlab = NULL, ylab = NULL,
                         title = NULL,
-                        symbolSize = c(4, 4, 4, 2),
+                        symbolSize = c(10, 10, 10, 4),
```

Comment:
How you determined the symbol size parameter, and similar parameters values?

Reply:
You can get the attribute values in Cx e.g sizes and other related values http://canvasxpress.org/api/general_attributes.html you can find here